app/routes/docs.notices.schema.mdx

```diff
@@ -12,47 +12,62 @@ export function loader() {
   return feature('SCHEMA') || new Response(null, { status: 404 })
 }
 
-# GCN Unified Schema
+# Unified Schema
 
-Introducing the Unified Schema, a standardized framework that simplifies the creation of new GCN Notice types across various missions, observatories, and instruments, incorporating both common and mission-specific properties. We employ JavaScript Object Notation (JSON) for these Notices, a versatile internet data format with minimal ties to JavaScript. JSON offers flexibility in data representation with few structural constraints. Meanwhile, JSON Schema serves as the standard for specifying the structure, data types, and validation rules for JSON documents.
+The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema).
 
-The process of crafting each mission schema involves the utilization of a set of core JSON schemas, forming the foundational building blocks for generating JSON-format Notices. This approach allows the flexibility of inclusion of mission-specific parameters while maximizing the use of the core schema. As a result, it ensures a consistent unit system for common properties across all missions.
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
```

Comment:
```suggestion
The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.
```
It looks like a single line paragraph with brackets. Other option is merge with next paragraph.